src/sentry/static/sentry/app/views/performance/transactionDetails/quickTrace.tsx

```diff
@@ -121,7 +121,23 @@ function QuickTracePills({
         organization={organization}
         events={[root]}
         text={t('Root')}
-        hoverText={t('View the root transaction in this trace.')}
+        hoverText={
+          <span>
+            <Truncate
+              value={root.transaction}
+              maxLength={30}
+              leftTrim
+              trimRegex={/\.|\//g}
+              expandable={false}
+            />
+            <br />
```

Comment:
ngl, this `<br />` is cause I wasn't sure how to split the name + duration apart in a reasonable way. Is there a better thing I could use here?